src/form/Select/Select.tsx

```diff
@@ -121,8 +121,10 @@ export interface SelectProps {
 
   /**
    * The list of KeyCodes for creating select values.
+   * The default is ['Enter']
+   * Typical options would be: ['Enter', 'Tab', 'Space', 'Comma']
    */
-  valuesSplitterKeyCodes?: string[];
+  selectOnKeys?: string[];
```

Comment:
Could we improve this type? Is there a keycode type the browser has we could use ?